### PR TITLE
Azure pipelines

### PR DESCRIPTION
**Pull request by thornbill**
**Changes**
Add Azure Pipelines config to run eslint and yarn package.

**Issues**
N/A

### CODE REVIEW

**Comment by EraYaN** [comment]
Alright how do you get it to fill in the WebVersion variable?

**Comment by thornbill** [comment]
@EraYaN Right now that is just set statically https://github.com/thornbill/jellyfin-web/blob/azure-pipelines/.ci/azure-pipelines.yml#L4. The tarball generated by `yarn pack` includes the version number from `package.json`. We aren't currently tracking the web version there so it just stays `0.0.0`. If you have any better ideas on how to pull that value, I am definitely open to that.

**Comment by EraYaN** [comment]
I mean you can just upload it with a wild card `*` ;) I do that with the setup as well.

**Comment by thornbill** [comment]
Right that makes perfect sense... let me fix that.

File: .npmignore [hunk 1]
```diff
@@ -1,7 +1,9 @@
+.editorconfig
 .gitattributes
 .github
 .drone.yml
 .eslintrc.yml
+azure-pipelines.yml
```
Comment (EraYaN) [inline .npmignore]: This should be `.ci/` maybe?

Comment (thornbill) [reply .npmignore]: Yeah I missed that when I moved it.

**Comment by thornbill** [comment]
Well it doesn't look like wildcards are supported in the `targetPath` for publishing artifacts. May need to add an intermediate copy task.

**Comment by EraYaN** [comment]
The setup is moved to the Artifacts Directory and then the whole thing is uploaded.

**Comment by anthonylavado** [comment]
This good?

**Comment by thornbill** [comment]
Yeah it should be all good.

**Pull request merged**